tauri-app/src/lib/components/tables/OrderAPY.test.ts

```diff
@@ -0,0 +1,90 @@
+import { render, screen, waitFor } from '@testing-library/svelte';
+import { test, vi } from 'vitest';
+import { expect } from '$lib/test/matchers';
+import { mockIPC } from '@tauri-apps/api/mocks';
+import type { OrderAPY } from '$lib/typeshare/subgraphTypes';
+import { QueryClient } from '@tanstack/svelte-query';
+import { formatUnits } from 'viem';
+import OrderApy from './OrderAPY.svelte';
+
+function formatApyToPercentage(value: string): string {
```

Comment:
why do we have an ad hoc function for this instead of a standard way to format numbers?

Reply:
moved to a relevant place and made it to be more general, so can be used elsewhere when there is a nee to convert a bigint to a percentage